test/topology/test_cluster_features.py

```diff
@@ -157,8 +158,11 @@ async def test_downgrade_after_successful_upgrade_fails(manager: ManagerClient)
     servers = await manager.running_servers()
     await change_support_for_test_feature_and_restart(manager, servers, enable=True)
 
+    # Workaround for scylladb/python-driver#230 - the driver might not
+    # reconnect after all nodes are stopped at once.
+    cql = await reconnect_driver(manager)
```

Comment:
Don't we need this in other tests though as well?
I can merge this PR, but it looks to me like it deserves a follow-up with adding this to other test cases too, which also restart the node.

Reply:
Right, there is also `test_joining_old_node_fails` which restarts all nodes at once, at the beginning.

Do we need to do it for other test cases? Others do restart nodes, but not all at once.

Reply:
It will be more interesting to do it only for `test_joining_old_node_fails` - and we'll see if other test cases are failing. If not, it could be more evidence that it's precisely restarting all nodes that causes issues in the driver.

Reply:
Followed up here: https://github.com/scylladb/scylladb/pull/14291